playbooks/prep-node-disks.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
The playbook name is really generic, and should be renamed to something more specific to the task.